lms/static/sass/views/_login-register.scss

```diff
@@ -339,8 +339,8 @@ $sm-btn-linkedin: #0077b5;
     .action-primary {
         @extend %btn-primary-blue;
         width: 100%;
-        height: ($baseline*2);
         margin-top: 1em; // Breathing room above (CR)
+        padding: ($baseline/2) 0; // Vertically aligns the text (CR)
```

Comment:
Even though the cases you are fixing have a full width, please add padding for the inline cases as well: 

```
padding: ($baseline/2) $baseline;
```

Also you can remove the comments on that and the line above.